frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -52,15 +52,22 @@ class ListDropdown_ extends React.Component {
       const { resources, dataFilter } = nextProps;
 
       state.items = {};
```

Comment:
Sorry for missing this nit, but we dont need this since we are pushing the `sortedList` into the `state.items` on the line#69